### PR TITLE
Update configfile.pp

### PR DESCRIPTION
**Pull request by sl0m0ZA**

### CODE REVIEW

File: manifests/configfile.pp [hunk 1]
```diff
@@ -61,3 +62,5 @@
     }
   }
 }
+
+#close #3
```
Comment (sl0m0ZA) [inline manifests/configfile.pp]: close #3

**Comment by sl0m0ZA** [comment]
close #3

**Comment by sl0m0ZA** [comment]
@trefzer I have added links to the documented examples here

**Comment by trefzer** [comment]
I don't thinks this is usefull. If one reads the text file and not the website (which at least I normaly do), it's not clear where the link goes. Instead using ::dovecot  it should be clear to find the documentation in the init.pp class.

**Comment by sl0m0ZA** [comment]
Links removed and other comments attended to

**Comment by trefzer** [comment]
looks good.
THanks

**Pull request merged**